tests/wicked/before_test.pm

```diff
@@ -191,7 +191,8 @@ EOT
     # Disable firewall by default.
     # Tests which need firewall, should explicit enable it
     if (script_run('systemctl is-active -q ' . opensusebasetest::firewall) == 0) {
-        systemctl("disable --now " . opensusebasetest::firewall);
+        systemctl("stop " . opensusebasetest::firewall);
```

Comment:
Just in case, I would wrote some comment here. That `disable --now` doesn't work with `SuSEfirewall2`, otherwise we might run in it again?

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/22026